### ISSUE TITLE
Docs: retraining pretrained models

### ISSUE THREAD

**Issue by rhine3** [kitzeslab/opensoundscape#970]
If we get the "500birds" model into OPSO, or another retrainable model (e.g. HawkEars?), add a standalone tutorial about how to retrain a model with various options (e.g. changing learning rate, freezing feature extraction layer)

Potentially also add a "train a shallow classifier from embeddings" tutorial as described by Sam here: #896
